dbt_coves/tasks/load/airbyte.py

```diff
@@ -44,7 +44,7 @@ def register_parser(cls, sub_parsers, base_subparser):
         subparser.add_argument(
             "--secrets",
             type=str,
-            help="Secret files location for Airbyte configuration",
+            help="Secret files location for Airbyte configuration, i.e. '/var/secrets'",
```

Comment:
This is not a great example. People will probably put secrets in the repo, not at system level. It should be something like `./secrets`